inc/thorvg.h

```diff
@@ -184,6 +184,30 @@ struct Matrix
 };
 
 
+/**
+ * @brief A data structure representing a texture mesh vertex
+ * 
+ * @param pt The vertex coordinate
+ * @param uv The texture coordinate in the range (0..width, 0..height)
```

Comment:
Could you please add ?  Once the feature is fully stable we will add the release tag.
@BETA_API

Reply:
In my opinion, the uv range would be better to use in the normalized range like (0 ~ 1). 
Otherwise, the user must get the size of the image first. We can skip that usage if it's possible.

Reply:
@hermet I originally [matched skia](https://github.com/rive-app/rive-cpp/blob/master/skia/renderer/src/skia_factory.cpp#L170), which uses 0 - width/height, but I agree that 0 - 1 is better and I will change it.